src/pages/Products/Products.jsx

```diff
@@ -1,22 +1,28 @@
 import "./products.css"
 import { Card, FilterSideNav } from "../../components";
+import { useProducts } from "../../context/product-context";
 
 const Products = () => {
+    const { availableProducts } = useProducts();
+
     return (
         <>
             <main className="prod-content">
                 <FilterSideNav />
                 <section className="prod-section">
-                    <h3>Products</h3>
+                    <h3>Showing Products { `${availableProducts.length} / ${availableProducts.length}` }</h3>
                     <div className="card-container">
-                        <Card />
-                        <Card />
-                        <Card />
-                        <Card />
-                        <Card />
-                        <Card />
-                        <Card />
-                        <Card />
+                        {
+                            availableProducts.map(({
+                                _id,
+                                title,
+                                price, 
+                                subtitle, 
+                                rating, 
+                                imgSrc
+                                }) => (
+                                <Card key={_id} price={price} title={title} subtitle={subtitle} rating={rating} imgSrc={imgSrc}/>
```

Comment:
can Pass **product** as prop to Card Component and destructure it in Card Component for easy syntax (Suggestion)